mod_deps.properties

```diff
@@ -1,7 +1,7 @@
 # mod_deps should be 'after:<mod identifier>' (unquoted, semicolon-separated, \escaped for newline, see example below)
-bopver=1.9.4-4.1.0.2037
+bopver=1.9.4-4.1.0.2048
```

Comment:
What was the need to change this? Nothing added to BoP after build 2037 should affect us, so there isn't much need to lock us to the absolute latest BoP build.